src/v2/client/startLoginFlow.js

```diff
@@ -38,7 +39,16 @@ export async function startLoginFlow(settings) {
     return emailVerifyCallback(settings);
   }
 
-  if (settings.get('useInteractionCodeFlow')) {
+  // do not call interact when flow=CONTINUE
+  // it should only continue existing flow or error
+  const configuredFlow = settings.get('flow');
+  if (configuredFlow && configuredFlow === Enums.CONTINUE_FLOW) {
+    const meta = await getSavedTransactionMeta(settings);
+    if (!meta) {
+      // TODO: throw error???
```

Comment:
Yes. The AuthJS implementation will throw an error something like this: 

https://github.com/okta/okta-auth-js/pull/1009/files#diff-b19423f759fc6861f7cadf71126b4dc150563ec3537d58d6814b13cd7bcee8b5